src/stories/QuestionCommentIcon.stories.tsx

```diff
@@ -13,8 +13,8 @@ export default {
 } as ComponentMeta<typeof QuestionCommentIcon>;
 
 const Template: ComponentStory<typeof QuestionCommentIcon> = (
-  args,
-  { globals: { locale } }
+    args,
```

Comment:
Why are those indentation changes still happening, and in other commits I saw you renaming ' --> " .

Reply:
Aren't we enforcing successful push requests by passing the pretyfier?

Reply:
ok we are solving this elsewhere, resolving